zuul_alpha/core.py

```diff
@@ -85,8 +84,19 @@ def _get_plaintext_private_key(
 
         return response[u'Plaintext']
 
+    class Encryptor:
```

Comment:
Ya know -- frankly I just don't like it. :P I actually thought it was against the style guide, but I've double checked and sure enough [its allowed](https://google.github.io/styleguide/pyguide.html#Nested/Local/Inner_Classes_and_Functions). If you want to keep it in here, go for it. I think the one negative is re: testing.